src/Tracy/Debugger.php

```diff
@@ -400,9 +407,10 @@ public static function setLogger(ILogger $logger)
 	public static function getLogger()
 	{
 		if (!self::$logger) {
-			self::$logger = new Logger(self::$logDirectory, self::$email, self::getBlueScreen());
+			self::$logger = new Logger(self::$logDirectory, self::$email, self::getBlueScreen(), self::$fromEmail);
```

Comment:
Ok, it does not need to be passed to Logger class but it must stay as property in Debugger class. (I am going to push new commit about this in few sec). Because when I want to configure fromEmail in neon from 'debugger' section it must be declared as static property in Debugger class, or am I wrong?